js/web/lib/wasm/jsep/webgpu/ops/fuse-utils.ts

```diff
@@ -17,16 +22,39 @@ export const getActivationSnippet = (attributes: InternalActivationAttributes, v
       return `value = (${valueType}(1.0) / (${valueType}(1.0) + exp(-value)));`;
     case 'Clip':
       return `value = clamp(value, ${valueType}(uniforms.clip_min), ${valueType}(uniforms.clip_max));`;
+    case 'HardSigmoid':
+      return `value = max(${valueType}(0.0), min(${valueType}(1.0), ${valueType}(uniforms.alpha) * value + ${
+          valueType}(uniforms.beta)));`;
     // TODO: adding other activations that can be fused.
     default:
       return '';
   }
 };
 
+export const appendActivationUniformsData =
+    (attributes: InternalActivationAttributes, programUniform: ProgramUniform[]) => {
+      if (attributes.activation === 'Clip') {
+        programUniform.push({type: 'float32', data: attributes.clipMax!}, {type: 'float32', data: attributes.clipMin!});
+      } else if (attributes.activation === 'HardSigmoid') {
+        programUniform.push({type: 'float32', data: attributes.alpha!}, {type: 'float32', data: attributes.beta!});
+      }
```

Comment:
Can we add an error or a warning if the activation is not supported, in the `else` block?

Reply:
We can't add an error here. `Clip` or `HardSigmoid` have two extra attributes. So we need to add uniforms for them. Some activations like `Relu/Sigmoid` don't have extra  attributes. So they don't need to add uniforms. But they are valid activations. However, your suggestion is good. I throw the error in L32 for un-supported activations.